### DISCUSSION TITLE
How to retrieve transactions via API?

### DISCUSSION THREAD

**Discussion by mykolas-s** [getsentry/support#53] [Support] [answered]
Hello,
I am looking for a way to retrieve all transactions from sentry API. I can get transactions by their IDs one by one from this endpoint - /api/0/projects/{organization_slug}/{project_slug}/events/{event_id}/

However, endpoint /api/0/projects/{organization_slug}/{project_slug}/events/ does not return any events of type "transaction", only of type "error".

Is there a way to retrieve all transactions via API?

**Comment by edwardgou-sentry** [comment, marked as answer]
Hi, if you're just looking for all transaction names within a timeframe, you can use the `events` api like this:

`https://sentry.io/api/0/{organization_slug}/{project_slug}/events/?statsPeriod=24h&field=transaction&field=count()&query=event.type:transaction`
The `event.type:transaction` query condition will ensure that only transaction events are returned from the query.
You can adjust the `statsPeriod` parameter to widen or shorten the time frame of your query.

For more information on how to use the `events` api, please check here:
https://docs.sentry.io/api/discover/query-discover-events-in-table-format/

Thanks!
